metadata/licenses/private/rules/license.bzl

```diff
@@ -0,0 +1,65 @@
+"""Declares rule `license`."""
+
+load("//:defs.bzl", "PackageAttributeInfo")
+
+def _license_impl(ctx):
+    metadata = ctx.actions.declare_file("{}.package-metadata.attribute.build.bazel.supply-chain.license.json".format(ctx.attr.name))
+
+    ctx.actions.write(
+        output = metadata,
+        content = json.encode({
+            "kind": "build.bazel.supply-chain.license",
+            "label": str(ctx.label),
+        }),
```

Comment:
Is there a reason that `kind` is needed in the file, or do you see it more as a convention that custom attributes should follow?

It seems like `kind` could always be inferred when reading the `PackageMetadataInfo` json file when getting the path to this json file.

Reply:
I think it's useful for debugging: you can look at an attribute file to see what should be in there